src/widgets/topicFrequencyGraph.js

```diff
@@ -209,8 +209,7 @@ export async function topicFrequencyGraphViaGoals(scopes) {
 
   topicsAndParticipants.forEach((goalData) => {
     goalData
-      .get('topics')
-      .forEach(({ topic, reportIds }) => {
+      .get('topics')?.forEach(({ topic, reportIds }) => {
         const topicResponce = topicsResponse
```

Comment:
How is this related to the other changes

Reply:
Looks like  a bad change will roll back.

Reply:
O wait this is because when you filter on RTR for a specific recipient and no results come back we need to check before accessing.